numpy/core/_internal.py

```diff
@@ -631,4 +631,17 @@ class TooHardError(RuntimeError):
     pass
 
 class AxisError(ValueError, IndexError):
-    pass
+    """ Axis supplied was invalid. """
```

Comment:
Probably, but it can be done in a separate PR as the imports also need to be changed. Maybe `_globals.py` should be renamed `_global_singletons` or `_global_exceptions` ;) There are other errors scattered about that maybe we should move at some point.